lib/rails_cloud_tasks/app_engine.rb

```diff
@@ -4,17 +4,23 @@
 module RailsCloudTasks
   module AppEngine
     class << self
+      def location_id
```

Comment:
This module concerns only App Engine related information.
I think the precedence of env-vars is better located in the Configuration class. What do you think?

Reply:
@guilhermearaujo done!